### PR TITLE
[SQL] Release patches archiving and concatenation

### PR DESCRIPTION
**Pull request by ridz1208**
## Brief summary of changes
Preparation for release 22.0

Concatenation of the patches was done using the following commands
```
mkdir SQL/Archive/22.0
git mv SQL/New_patches/*.sql SQL/Archive/22.0/
for n in SQL/Archive/22.0/*.sql; do echo $n; echo -e "\nSELECT 'Running: $n';\n" >> SQL/Release_patches/21.0_To_22.0_upgrade.sql; cat $n >> SQL/Release_patches/21.0_To_22.0_upgrade.sql; done;```

### CODE REVIEW

File: SQL/Release_patches/21.0_To_22.0_upgrade.sql [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+SELECT OrderNumber INTO @ordernumber FROM LorisMenu WHERE Label='Imaging';
+UPDATE LorisMenu SET OrderNumber=(OrderNumber+1) WHERE Parent IS NULL AND OrderNumber>@ordernumber;
+
+INSERT INTO LorisMenu (Label, OrderNumber)
+SELECT 'Genomics', OrderNumber+1 FROM LorisMenu WHERE Label='Imaging';
+
+SELECT ID INTO @parentid FROM LorisMenu WHERE Label='Genomics';
+UPDATE LorisMenu SET Parent=@parentid WHERE Link='genomic_browser/';
+
+UPDATE permissions SET Code='quality_control', description='Quality Control access' WHERE code='data_team_helper';
```
Comment (johnsaigle) [inline SQL/Release_patches/21.0_To_22.0_upgrade.sql]:
```suggestion
UPDATE permissions SET Code='quality_control', description='Quality Control access' WHERE code='data_team_helper';
```

Should we make a ticket to create a unique permission for this?

Comment (johnsaigle) [reply SQL/Release_patches/21.0_To_22.0_upgrade.sql]: Also maybe the description here should say Behavioural/Imaging QC depending on which permission it actually is. It's not clear from here.

**Comment by johnsaigle** [comment]
Are Raisinbread updates going in a separate PR?

File: SQL/Release_patches/21.0_To_22.0_upgrade.sql (hunk 1 above)
Comment (ridz1208) [reply SQL/Release_patches/21.0_To_22.0_upgrade.sql]: it's both.

the reason is that whoever coded the imaging_qc module used the `data_team_helper` permission to control access to it. I renamed the permission to `quality_control` to be more generic and yes I do think they shou;d have each their own permissions it was just out of scope to do it  in that PR.

**Comment by ridz1208** [comment]
@johnsaigle I am working right now on updating RB

**Comment by johnsaigle** [comment]
> 1) DicomArchiveTestIntegrationTest::testdicomArchiveFilterClearBtn
UnexpectedJavascriptException: TypeError: input is null

DICOM archive transitory test failure. I'll restart Travis

**Comment by PapillonMcGill** [comment]
ran `21.0_To_22.0_upgrade.sql` and no reported error. Did a few test on the web interface and Ok.

**Pull request merged**